pkg/metrics/metrics.go

```diff
@@ -1204,6 +1209,17 @@ func CreateConfiguration(metricsEnabled []string) (Configuration, []prometheus.C
 			collectors = append(collectors, FQDNActiveIPs)
 			c.FQDNActiveIPs = true
 
+		case Namespace + "_" + SubsystemFQDN + "_regex_lru_accesses_total":
+			FQDNRegexLRUAccesses = prometheus.NewCounterVec(prometheus.CounterOpts{
+				Namespace: Namespace,
+				Subsystem: SubsystemFQDN,
+				Name:      "regex_lru_accesses_total",
+				Help:      "Keeps track of the total number of hits and misses to the FQDN regex LRU",
+			}, []string{LabelOutcome})
```

Comment:
I gave this a spin on my dev stack and the thing I found a bit weird is the `outcome` here being either `fail` or `success` which would make me believe a failure happened without knowing what this PR is doing. Maybe just create a separate label with `hit`/`miss` values?

Reply:
Implemented.